socorro/processor/processor_2015.py

```diff
@@ -257,33 +294,17 @@ def process_crash(self, raw_crash, raw_dumps, processed_crash):
         # for backwards compatibility:
         processed_crash.completeddatetime = completed_datetime
 
-        self._log_job_end(
-            processed_crash.success,
+        self.config.logger.info(
+            "finishing %s job: %s",
+            'successful' if processed_crash.success else 'failed',
```

Comment:
As a side note, when I'm reading processor logs, I get tripped up by this message. It's not uncommon for me to be processing a crash, see this message, assume it means that the crash is completely through the processor including crash storage, kill the processor, and then discover later that the crash hadn't been stored, yet. So then I have to start up the processor and process it again and wait some more.

It'd help me to clarify this logging statement and the other one as being specific to the transform step and not all of processing.

Does that sound prudent? Maybe "starting transform for crash: %s" and "finished %s transform for crash: %s"?

Reply:
We can kick this discussion to another PR. It shouldn't hold up this one. And if it doesn't affect anyone else, that's cool--we can leave it out altogether.

Reply:
I'm on board. Both places are modified to use the 'starting transform' logic

Reply:
I appreciate it!